packages/ceramic-common/src/doctype.ts

```diff
@@ -28,142 +28,144 @@ export enum AnchorStatus {
  * Describes anchor record
  */
 export interface AnchorRecord {
-    prev: CID; // should be CID type
-    proof: CID; // should be CID type
-    path: string;
+  prev: CID; // should be CID type
```

Comment:
is there a reason for tab size change in the doc? it's hard to see the changes you made

Reply:
I just used my editors default. However the eslint configuration that Paul set up (which we should slowly start to adopt everywhere) uses 2 for tabwidth :)
https://github.com/3box/eslint-config-3box

Reply:
let's create a separate issue for formatting based on the `eslint-config-3box` and reformat the whole project :)